src/Tempest/Console/src/Commands/Generators/MakeControllerCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Commands\Generators;
+
+use Tempest\Console\ConsoleArgument;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Console\Stubs\ControllerStub;
+use Tempest\Generation\HasGeneratorCommand;
+
+final class MakeControllerCommand
+{
+    use HasGeneratorCommand;
+
+    #[ConsoleCommand(
+        name       : 'make:controller',
+        description: 'Create a new controller class with a basic route',
+        aliases    : ['controller:make', 'controller:create', 'create:controller'],
+    )]
+    public function __invoke(
+        #[ConsoleArgument(
+            help: 'The name of the controller class to create ( "Controller" will be suffixed )',
+        )]
+        string $className,
+        #[ConsoleArgument(
+            help: 'The route path inside the controller',
+        )]
+        ?string $controllerPath = null,
+        #[ConsoleArgument(
+            help: 'The view name to return in the controller',
+        )]
+        ?string $controllerView = null,
+    ): void {
+        $suggestedPath = $this->getSuggestedPath(
+            className  : $className,
+            pathPrefix : 'Controllers',
+            classSuffix: 'Controller',
+        );
+        $targetPath = $this->promptTargetPath($suggestedPath);
+        $shouldOverride = $this->askForOverride($targetPath);
+
+        $this->stubFileGenerator->generate(
```

Comment:
Maybe not within the scope of this PR, but it might be nice to also make a view file together with the controller or something alike.

Reply:
I'll dig this idea, but I think this could be a flag or something to additionally create the view 🤷 
And yes maybe in a separate PR as an improvement of the `make:controller` method.
As this one mainly aim as setup generator commands and with basic examples.